### ISSUE TITLE
[BUG] Import certificate with dcmimport

### ISSUE THREAD

**Issue by ghost** [ThePrez/DCM-tools#23]
Importing a certificate store containing the CAs and the personal/server certificate (which i want to use for a http server) does not do the correct thing.
The personal certificate from the source keystore is importet as a CA certificate.
And also the parameter --cert is not treaten correctly. The label is just the one used by the source keystore.

When the server certificate is importet as a CA certificate you are unable to assign it to dcm applications.

If i import my source keystore manually via DCM GUI the certificate is correctly importet as a server certificate.

**Comment by spirjade** [comment]
Yeah. I faced same issue. With DCMIMPORT, server certificate is always imported as CA certificate.Then later we are not able to assign to HTTP SERVER. this is blocking.if you could please advise,that will be of great help for us .thanks

**Comment by iangioffre** [comment]
I am facing the same issue and there isn't a way around it that I have found. Would very much appreciate an update on this.
